tests/test_main.py

```diff
@@ -708,6 +711,9 @@ def test_module_files(self):
             if (
                     # The sentinel actually appearing in code messes things up
                     'executing' in filename
+                    # ignore pytest files because they break the SentinelNodeFinder
+                    or '_pytest' in filename
```

Comment:
I think this file stopped being a problem once annotations were considered in 3.6+

Reply:
no problem any more.

Reply:
So these lines should be removed, right?

Reply:
sorry you are right. I forgot that.

Reply:
Ah, it just hit a genuine failure because it doesn't know how to distinguish between the comprehension codes in [`{t[0] for t in lines2} - {t[0] for t in lines1}`](https://github.com/pytest-dev/pytest/blob/2be1b8f3559570c456f4bf64fea8067e368dcdfd/src/_pytest/pytester.py#L172). This is covered in the normal tests. So `pytester.py` specifically should be excluded.

Reply:
sigh, another pypy optimisation. can you skip if `PYPY and '__debug__' in source.text'`?

if it fails again you can go back to skipping all pytest files